cpp/src/centrality/betweenness_centrality.cuh

```diff
@@ -17,7 +17,7 @@
 // Author: Xavier Cadet xcadet@nvidia.com
 
 #pragma once
-#include <rmm/thrust_rmm_allocator.h>
+#include <rmm/device_vector.hpp>
```

Comment:
```device_vector``` is not used in this .cuh file but used in betweenness_centrality.cu file. So, better move this include statement to the .cu file.